### ISSUE TITLE
missing dependency after install : pooch

### ISSUE THREAD

**Issue by MrFuppes** [imk-toolkit/imk-toolkit#39]
since `pooch` is required to run the [usage](https://github.com/imk-toolkit/imk-toolkit#usage) script, should this be a normal dependency? In the current [dependencies specs](https://github.com/imk-toolkit/imk-toolkit/blob/master/imktk/pyproject.toml), it's dev-only.

Details: after package installation, running 
```
t = xr.tutorial.open_dataset("rasm").load().Tair
```
raises
> ModuleNotFoundError: No module named 'pooch'
(Python 3.9.4 on Windows 10)

**Comment by ucyo** [comment]
Good catch! 👍🏼 
This dependency is only relevant if we want to use this tutorial dataset. We have following options:

1. Add `pooch` to dependency graph and move on
2. Change the example code to something similar to  `var = xr.open_dataset(inputfile).variable`
3. Create a small test dataset and distribute it with the repository

It seems to me that option 3 is the most future-proof since it reduces the dependency graph compared to the other options and opens the possibility to use that example file in tests. @MrFuppes What do you think?

**Comment by MrFuppes** [comment]
Jup, #3 seem most reasonable to me as well. We might also want to have such demo data for e.g. a plot example.

Regarding data for tests, I guess those would not have to be included in the build.

**Comment by ucyo** [comment]
I requested a small ERA5 dataset which we might be able to use from [Copernicus](https://cds.climate.copernicus.eu). But it takes a bit of time to make it available. I yet have to get access to it...

**Comment by ucyo** [comment]
I now have the data downloaded. I will look into the license and check if it is possible to upload the data. If not, I will analyse the data distribution of the variables and generate fake data 👀

**Comment by MrFuppes** [comment]
`xarray` can use data via OPeNDAP, no? Could that be an option?

**Comment by ucyo** [comment]
Yes. OPeNDAP could also be an option. Do you know of a open accessible source we could use?
